rendered/draft-hopwood-coinbase-balance.html

```diff
@@ -1,129 +1,10 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <title>Draft hopwood-coinbase-balance: Blocks should balance exactly</title>
-    <meta charset="utf-8" />
-<meta name="viewport" content="width=device-width, initial-scale=1"><link rel="stylesheet" href="css/style.css"></head>
+    <title>ZIP 236: Blocks should balance exactly (redirect)</title>
+    <meta http-equiv="refresh" content="0; zip-0236">
```

Comment:
Now that I think about it, this won't work locally (but a bunch of other links don't work in a local checkout either). It's not urgent, I'll improve it when I have time.